statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -3225,11 +3229,16 @@ def test_serial_correlation(self, method, lags=None):
                     FutureWarning
                 )
 
+            if df_adjust:
+                df_model = self.df_model - 1
+            else:
```

Comment:
Skip the else by initially setting model_df = 0, and then only have the if clause.

Reply:
Done.